Sources/MessagingInApp/Views/InAppMessageView.swift

```diff
@@ -38,6 +38,13 @@ public class InAppMessageView: UIView {
         }
     }
 
+    // Inline messages that have already been shown by this View instance.
+    // This is used to prevent showing the same message multiple times when the close button is pressed.
+    //
+    // When persistent vs non-persistent messages and metrics features are implemented in the SDK, this array may be
+    // replaced with a global list of shown messages.
+    var previouslyShownMessages: [Message] = []
```

Comment:
Call-out to the comments for this property. 

When persistent vs non-persistent messages and metrics features are implemented for inline in-app messages, I am expecting that this property will be replaced. 

For modal messages today, once a message is shown, the SDK tracks a metric and doesn't show the same messages again. I imagine that inline will behave similarly.